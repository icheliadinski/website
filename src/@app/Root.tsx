import React from 'react'
import { BrowserRouter, Switch, Route, Redirect } from 'react-router-dom'
import { UnderConstruction } from '@pages/under-construction'

export const Root: React.FC = () => {
  return (
    <BrowserRouter>
      <Switch>
        <Route path="construct" component={UnderConstruction} />
        <Redirect to="construct" />
      </Switch>
    </BrowserRouter>
  )
}
