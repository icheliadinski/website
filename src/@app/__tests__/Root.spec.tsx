import React from 'react'
import { shallow, ShallowWrapper } from 'enzyme'
import { Root } from '../Root'

describe('[Root]', () => {
  let wrapper: ShallowWrapper

  beforeEach(() => {
    wrapper = shallow(<Root />)
  })

  it('renders successfully', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
