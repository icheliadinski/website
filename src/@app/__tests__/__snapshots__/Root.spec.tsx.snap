// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Root] renders successfully 1`] = `
<BrowserRouter>
  <Switch>
    <Redirect
      to="/construct"
    />
    <Route
      exact={true}
      path="/"
    />
    <Route
      path="/construct"
    />
  </Switch>
</BrowserRouter>
`;
